madcad/prelude.py

```diff
@@ -0,0 +1,24 @@
+"""
+lazy imports for common utilites, globals and containers
```

Comment:
Ideally, the root module `madcad` already serve that purpose. It exposes most of the end-user functions. 
From the user perspective there is not need of an other exposing-only module, except eventually if we add one to expose only types and no functions.
From the developer perspective one more thing is to consider:  **the module loading order**.
At contrary to compiles languages, python is loading modules by executing their code. So imagine the following scenario:
```python
# the user is loading madcad for the first time
import madcad # this will start loading of madcad/__init__.py

# madcad/__init__.py
from .draft import draft   # this actually pause the loading of __init__.py to load draft.py and pick its content

# madcad/draft.py
from madcad import ...   # this finds __init__.py already initialized but partially loaded, so part of its content is not there yet.
```
For that reason, the madcad submodules should not import from the root module like in that prelude file, but only from other submodules.

Reply:
what i want as developer is be able to import some commonly used "core" stuff to reduce boilerplate code inside test_code and the modules that are not "core". I did not write the prelude.py with the end user in mind